pkg/apis/build/v1alpha2/build.go

```diff
@@ -142,9 +142,10 @@ func (b *Build) NotaryV1Config() *corev1alpha1.NotaryV1Config {
 }
 
 func (b *Build) DefaultProcess() string {
-	if b.Spec.DefaultProcess == "" {
-		return "web"
+	if b == nil {
```

Comment:
I think this nil check is not necessary now especially that this method has no logic.

Reply:
Following the pattern of the other functions in `build.go`. For example `NotaryV1Config` on line 137 has a similar setup just in case a nil build object gets passed in. 

I agree it's probably unnecessary though but for the sake of consistency maybe we should still keep it.

Reply:
Lets clean up other methods vs reimplementing other logic for consistency

Reply:
👍